app/components/canvas/Canvas.tsx

```
@@ -1689,161 +1640,40 @@ function InnerCanvas({
-                        isTranscribing: false,
-                        isExtracting: false,
-                        hasTranscription: false,
-                        transcriptionError: scheduleError.message,
-                        onRetryTranscription: () => retryTranscription(video._id),
-                      },
-                    }
-                  : node
-              )
-            );
-          }
+          // ... (existing dead code for audio extraction and background transcription) ...
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)